### PR TITLE
Create GHRC Portal

### PR DESCRIPTION
**Pull request by anupamdahal**
# Overview
Adding GHRC portal to EDS

#### Files Added

- protals/ghrc/images/ghrc-logo-blue-transparent.png
- protals/ghrc/images/ghrc-logo-dark.png
- portals/ghrc/config.json
- portals/ghrc/scripts.js
- portals/ghrc/styles.scss

### What areas of the application does this impact?

Portals. New Portal Created

List impacted areas.

- /portals
- /protals/ghrc/search

image

### CODE REVIEW

File: portals/ghrc/config.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+    "hasStyles": true,
+    "hasScrpits": false,
+    "logo":{
+        "id": "ghrc-logo",
+        "link": "https://ghrc.nsstc.nasa.gov/home",
+        "tittle": "GHRC Home"
```
Comment (abbottry) [inline portals/ghrc/config.json:7]: Typo here, should be `title`

**Review by abbottry — CHANGES_REQUESTED** [review]

File: portals/ghrc/config.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    },
+    "org": "GHRC",
+    "pageTittle": "GHRC Portal",
```
[the remark this answers is not in the document]
Comment (anupamdahal) [reply portals/ghrc/config.json]: Changes made in the latest commit "Fix discovered typos"

File: portals/ghrc/config.json (hunk 1 above)
Comment (anupamdahal) [reply portals/ghrc/config.json]: Changes made in the latest commit "Fix discovered typos"

File: portals/ghrc/config.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        "id": "ghrc-logo",
+        "link": "https://ghrc.nsstc.nasa.gov/home",
+        "title": "GHRC Home"
+    },
+    "org": "GHRC",
+    "pageTitle": "GHRC Portal",
+    "parentConfig": "edsc",
+    "query": {
+        "dataCenter": "NASA/MSFC/GHRC",
+        "hasGranulesorCwic": null
```
Comment (abbottry) [inline portals/ghrc/config.json:14]: Another typo: should be `hasGranulesOrCwic`

File: portals/ghrc/config.json [hunk 4]
```diff
@@ -1,20 +1,21 @@
...
+  "parentConfig": "edsc",
+  "query": {
+    "dataCenter": "NASA/MSFC/GHRC",
+    "hasGranulesOrCwic": null
+  },
+  "ui": {
+    "showOnlyGranulesCheckbox": false,
+    "showNonEosdisCheckbox": false
+  }
+  }
```
Comment (abbottry) [inline portals/ghrc/config.json:20]: It looks like you changed all of your tabs/spaces for the entire PR which.. is fine but the last closing brace should not be indented here.

**Review by macrouch — APPROVED** [review]

File: portals/ghrc/config.json (hunk 4 above)
Comment (anupamdahal) [reply portals/ghrc/config.json]: done! thank you!

File: portals/ghrc/config.json (hunk 3 above)
Comment (anupamdahal) [reply portals/ghrc/config.json]: Resolved!

**Review by abbottry — APPROVED** [review]

**Pull request merged**